### PR TITLE
add guide for running the visualization

### LINKED ISSUE #375: add docs for setting up vis dev environment

### PR DESCRIPTION
**Pull request by dyc3**
closes #375

### CODE REVIEW

File: data/dev-env/balancer.docker.toml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+api_key = "<randomly generate this>"
```
Comment (Victor-M-Giraldo) [inline data/dev-env/balancer.docker.toml:1]: Probably specify that this should be alphanumerical only so Chris/Mike don't run into the same problem I did.

**Comment by Victor-M-Giraldo** [comment]
I don't think the collector is working on my end.

**Comment by dyc3** [comment]
@Victor-M-Giraldo elaborate?

**Comment by Victor-M-Giraldo** [comment]
Sure. Commands ran are:

`docker-compose -f docker/with-balancer.docker-compose.yml up -d`
`yarn workspace ott-vis server`

I visit localhost:8080/state and get nothing. I think I forgot to enable load balancing though, so that's probably the issue. I'll comment again if it doesn't work

**Comment by Victor-M-Giraldo** [comment]
Still can't get it to work. I'll post my config files here.

balancer.toml
```toml
api_key = "a83Hec84qRCi"

[discovery]
method = "dns"
query = "opentogethertube"
service_port = 3002
```

collector.toml
```toml
collect_interval = "5s"

balancer_api_key = "a83Hec84qRCi"

[discovery]
method = "dns"
query = "balancer"
service_port = 8081
```

base.toml
```toml
[balancing]
enabled = true
```

**Comment by dyc3** [comment]
That looks right to me

**Review by Victor-M-Giraldo — APPROVED** [review]

**Review by moreno-michael — APPROVED** [review]
docker version working, no docker version not working on my machine

**Pull request merged**